source/common/config/grpc_mux_subscription_impl.h

```diff
@@ -56,8 +56,9 @@ class GrpcMuxSubscriptionImpl : public Subscription<ResourceType>,
     stats_.update_success_.inc();
     stats_.update_attempt_.inc();
     stats_.version_.set(HashUtil::xxHash64(version_info));
-    ENVOY_LOG(debug, "gRPC config for {} accepted with {} resources: {}", type_url_,
-              resources.size(), RepeatedPtrUtil::debugString(typed_resources));
+    ENVOY_LOG(debug, "gRPC config for {} accepted with {} resources with version {}", type_url_,
+              resources.size(), version_info);
+    ENVOY_LOG(trace, "resources: {}", RepeatedPtrUtil::debugString(typed_resources));
```

Comment:
Since multiple threads can be logging at the same time, I'd actually have a more specific line when printing out resources in the trace. Something like below:

```cpp
ENVOY_LOG(trace, "_gRPC config accepted with resources: {}_", RepeatedPtrUtil::debugString(typed_resources));
```

Reply:
I think demuxing multiple threads is a consumer issue, for example I have a log viewer that can colorize based on TID. So, no need to force it into a single statement.